web-registry/src/lib/tracker/useTracker.ts

```diff
@@ -0,0 +1,17 @@
+import { useTrack } from 'use-analytics';
+
+import { Buy2Event, LoginEvent } from './types';
+
+interface UseTracker {
+  track: (
+    eventName: string,
+    payload?: any | LoginEvent | Buy2Event,
+    options?: any,
+    callback?: (...params: any[]) => any,
+  ) => Promise<any>;
+}
```

Comment:
@wgwz some comments on this interface. Here you are mimicking the original interface of the library, but that is too open (unspecific) for our use cases.

I think that at the moment we are compulsorily informing the first two parameters (event name and payload). Therefore, they should be the only properties of the track function that we are wrapping at the moment, and the two mandatory ones.

The first, eventName, could be a type of the following form:
```
type EventName = 'login' | 'buy1' | 'buy2';
```
The Payload could be as follows:
```
type EventPayload = LoginEvent | Buy1Event | Buy2Event;
```
Finally, we would have a restrictive interface, where any name or payload event is not valid:
```
interface UseTracker {
   tracks: (
     eventName: EventName,
     payload: EventPayload
   ) => Promise<any>;
}
```